src/Core/Domain/Store/Exception/CannotToggleStoreStatusException.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Store\Exception;
+
+/**
+ * Thrown when cannot toggle store status
+ */
+class CannotToggleStoreStatusException extends StoreException
+{
+    /**
+     * Thrown when cannot toggle single store status.
+     */
+    public const SINGLE_TOGGLE = 10;
+
+    /**
+     * Thrown when cannot bulk toggle store status.
+     */
+    public const BULK_TOGGLE = 20;
```

Comment:
I don't see this one used for now Do you plan on adding bulk toggle in this PR as well or is it to anticipate a future one? Usually we try to limit the PR's content to what is essentially needed, this avoids adding dead code or unused code